spec/e2e/step_definitions/short_form_steps.coffee

```diff
@@ -138,6 +138,25 @@ module.exports = ->
       elem.isDisplayed()
     ).first().click()
 
+  @When /^I upload a "([^"]*)" as my proof of preference$/, (documentType) ->
+    # open the proof option selector and pick the indicated documentType
+    element.all(By.id("liveInSf_proof_option")).filter((elem) ->
+      elem.isDisplayed()
+    ).first().click()
+    element.all(By.cssContainingText('option', documentType)).filter((elem) ->
+      elem.isDisplayed()
+    ).first().click()
+    filePath = '/Users/mattluedke/Desktop/projects/dahlia/sf-dahlia-web/public/images/arrow-down.png'
```

Comment:
Ok. I'll at least leave a comment in the test so it's clear that the file upload itself is not important to succeed.